### ISSUE TITLE
[FEAT] Notify Breaking Features or Provide Changelog

### ISSUE THREAD

**Issue by czhang03** [secureblue/secureblue#353]
**Describe the benefit this feature would provide to the project**
Secureblue has several radical security measures like disable xwayland and bluetooth. Although these features are great for security, likely won't impact the work flow for many, and have ways to be disabled. They still impose burdens on people using secureblue for their daily work. 

For example, when secureblue disabled xwayland, my zoom wouldn't launch, and I have a meeting in the morning. I have to roll back, then debug to notice that xserver is missing, and scroll through ujust commands to find the toggle.

I think having a changelog/breaking feature via yafti would be a great idea as it helps with troubleshooting the problem people have been having, instead of them scrolling through the commit on GitHub.

**Describe the solution you'd like**
Use yafti to provide changelogs or a list of breaking changes, although I am not sure how to implement it.
We might need the following:
- a parsable (yaml, json, etc.) file for changelog/breaking change 
- let yafti log the date that it is last launched.
Check whether there are breaking changes from the last launch via the parsable file.

Again, it seems like yafti is pretty limited in its functionality as of now. I don't know if it can do complex logic like this.

**Describe alternatives you've considered**
- A `ujust news` command in the terminal would also be great, but it would require people to go into the terminal. It is much easier to implement than yafti, and less intrusive. 
- The `ujust changelogs` command will show change log of the packages, not the OS itself. 

**Declaration**
- [ x ] I declare that I have read the README and my feature request is in-scope.

Specifically, the readme declares:

> Avoid sacrificing usability for most use cases where possible

I think this is a usability issue, and the fix won't impact security.

**Comment by RoyalOughtness** [comment]
Agreed, this is long overdue

**Comment by RoyalOughtness** [comment]
@czhang03 what about something like what dotnet does: https://github.com/dotnet/announcements

they have a separate repo for announcements, that users can click "watch" to get emailed every time an urgent announcement is added

**Comment by czhang03** [comment]
This is certainly the easiest way to implement this feature, however, this would require an GitHub account, which not every user might have. It also relies on GitHub's ability to deliver timely emails to make sure the breaking changes arrives after the email, I haven't used to watch feature that much, thus I cannot comment on the punctuality of GitHub email.

However, I would be okay with doing this as a fallback and move to a more principled way to handle changelog in the future.

**Comment by RoyalOughtness** [comment]
@czhang03 The github email feature is immediate, but overall I agree with you. yafti would be preferable. It's currently being rewritten with new functionality. Once the rewrite is complete I can use it to add announcements like this

**Comment by czhang03** [comment]
@qoijjj Do you know where the rewrite is happening? The repo https://github.com/ublue-os/yafti seems pretty stale.

**Comment by RoyalOughtness** [comment]
@czhang03 I asked Kyle earlier today where the repo is, will let you know when I hear back

**Comment by RoyalOughtness** [comment]
the thread, for reference: https://discord.com/channels/1072614816579063828/1084120222400774325/1268620854628061265

**Comment by RoyalOughtness** [comment]
https://github.com/ublue-os/yafti/pull/229

**Comment by RoyalOughtness** [comment]
completed was accidental :smile:

**Comment by czhang03** [comment]
Thank you for letting me know about the rewrite. I will see how it goes. And if I have time, I will probably try to implement a changelog feature.

However, I will be really busy next year, and I also have several of my projects that are in desperate need of maintenance. I would love to help the ublue-based community in general, I just don't know how much time I can spare.

**Comment by RoyalOughtness** [comment]
https://github.com/secureblue/secureblue/commit/327eb2279d5269bf9a6e6b95960d02fff619dc44

**Comment by RoyalOughtness** [comment]
The yafti rewrite isn't going to be done any time soon, so for now I will simply be more diligent about minor release versions.

I recommend users subscribe to releases using this: 
![image](https://github.com/user-attachments/assets/fdeca485-4b0d-46aa-bf8b-ccfd06de69cb)

**Comment by RoyalOughtness** [comment]
https://github.com/secureblue/secureblue/blob/live/FAQ.md#how-do-i-get-notified-of-secureblue-changes

**Comment by RoyalOughtness** [comment]
closing in favor of: https://github.com/secureblue/secureblue/issues/381
